src/features/codeLensProvider.ts

```diff
@@ -188,8 +191,8 @@ function createCodeLensesForElement(element: Structure.CodeElement, fileName: st
             let range = element.Ranges[SymbolRangeNames.Name];
 
             if (range && testFramework && testMethodName) {
-                results.push(new RunTestsCodeLens(range, fileName, /*isTestContainer*/ false, testFramework, [testMethodName]));
-                results.push(new DebugTestsCodeLens(range, fileName, /*isTestContainer*/ false, testFramework, [testMethodName]));
+                results.push(new RunTestsCodeLens(range, fileName, element.DisplayName,/*isTestContainer*/ false, testFramework, [testMethodName]));
```

Comment:
Does TS have the equivalent of C# named arguments? It would be neat to use those instead of the comment...

Reply:
There is object destructuring , but should we do that here https://github.com/Microsoft/TypeScript/issues/467#issuecomment-404408760

Reply:
That's interesting approach...